GPU/GPUTracking/DataTypes/GPUTRDTrack.h

```diff
@@ -106,13 +107,15 @@ class GPUTRDTrack_t : public T
   GPUd() void setIsStopped() { mFlags |= (1U << kStopFlag); }
   GPUd() void setIsAmbiguous() { mFlags |= (1U << kAmbiguousFlag); }
   GPUd() void setChi2(float chi2) { mChi2 = chi2; }
+  GPUd() void setIsPadrowCrossing(int iLayer) { mIsPadrowCrossing |= (1U << iLayer); }
 
   // conversion to / from HLT track structure (only for AliRoot)
   GPUd() void ConvertTo(GPUTRDTrackDataRecord& t) const;
   GPUd() void ConvertFrom(const GPUTRDTrackDataRecord& t);
 
  protected:
   float mChi2;                      // total chi2.
+  unsigned char mIsPadrowCrossing;  // bits 0 to 5 indicate wheter track was in between two padrows
   unsigned int mRefGlobalTrackId;   // raw GlobalTrackID of the seeding track (either ITS-TPC or TPC)
   int mAttachedTracklets[kNLayers]; // indices of the tracklets attached to this track; -1 means no tracklet in that layer
   short mCollisionId;               // the collision ID of the tracklets attached to this track; is used to retrieve the BC information for this track after the tracking is done
```

Comment:
Could you move mCollisionID before mAttached..., to not lose 2 bytes on the alignment?

Reply:
Sure, did not know that alignment was a concern here. Sorry.
But from my very limited understanding of the topic, the situation is like this:

```
// size = 36, alignment = 4
// before:  +----+----+-*6-+----+
// members: |aaaa|bbbb|c**c|dde.|
// . represents a byte of padding
// a=mChi2, b=mRefGlobalTrackId, c=mAttachedTracklets, d=mCollisionId, e=mFlags
```

```
// size = 40, alignment = 4
// mine:    +----+----+----+-*6-+----+
// members: |aaaa|p...|bbbb|c**c|dde.|
// p=mIsPadrowCrossing
```

```
// size = 44, alignment = 4
// yours:   +----+----+----+----+-*6-+----+
// members: |aaaa|p...|bbbb|dd..|c**c|e....|
```

Maybe better is:
```
// size = 36, alignment = 4
// mine:    +----+----+-*6-+----+
// members: |aaaa|bbbb|c**c|ddep|
```
e.g. moving mIsPadrowCrossing after mFlags.

At least that what I get from testing in an interactive root session and defining
struct foo {...} and playing with alignof() and sizeof() on x86, not sure if this is valid since this is run on a GPU?

Reply:
I think this was just a typo. Putting `mIsPadrowCrossing` after `mFlags` is fine

Reply:
Done.